src/System.Private.CoreLib/shared/System/Diagnostics/Debug.cs

```diff
@@ -140,15 +174,14 @@ public static void Fail(string message, string detailMessage)
             Assert(false, message, detailMessage);
         }
 
-        private static string FormatAssert(string stackTrace, string message, string detailMessage, string indentString)
+        private static void WriteAssert(string stackTrace, string message, string detailMessage)
         {
-            string newLine = indentString + Environment.NewLine;
-            return SR.DebugAssertBanner + newLine
-                   + SR.DebugAssertShortMessage + newLine
-                   + message + newLine
-                   + SR.DebugAssertLongMessage + newLine
-                   + detailMessage + newLine
-                   + stackTrace;
```

Comment:
Fixes another existing bug in Debug.Fail indentation.
We didn't really need to indent end of each line with indent string.